modules/juce_core/javascript/juce_Javascript.cpp

```diff
@@ -1549,6 +1550,35 @@ struct JavascriptEngine::RootObject   : public DynamicObject
 
             return var::undefined();
         }
+        
+        static var splice (Args a) {
+            if (Array<var>* remainArray = a.thisObject.getArray())
+            {
+                int startIndex = get (a, 0).isUndefined() || get (a, 0).isVoid() ? 0 : (int) get (a, 0);
+                if (startIndex < 0)
+                {
+                    startIndex = remainArray->size()+startIndex;
+                }
+
+                int numel = a.numArguments > 1 ?  (int) get (a, 1) : remainArray->size()-startIndex;

```

Comment:
Of course. It passes all unit tests, which cover all aspects of behaviour defined in the standard. Your comment implies that this will not run, but I think you're just looking at the 2nd half of the statement.